botnim/collect_sources.py

```diff
@@ -36,18 +43,249 @@ def collect_sources_google_spreadsheet(context_name, source):
             file_streams.append((f'{context_name}_{idx}.md', io.BytesIO(content.strip().encode('utf-8')), 'text/markdown'))
     return file_streams
 
-def collect_context_sources(context_, config_dir: pathlib.Path):
```

Comment:
I have no idea what's going on here.
Why are we checking for a 'sources' key in the config all of a sudden? 
Why do we need two `collect_sources_google_spreadsheet` methods??

Reply:
I removed the multiple resources handling.